packages/web/src/scss/components/TextField/README.md

```diff
@@ -132,10 +132,29 @@ forget to change not only the input type but also `aria-pressed` and
 ## Validation States
 
 Validation states can be presented either by adding a CSS modifier class
-(`TextField--success`, `TextField--warning`, `TextField--error`), or by adding
+(`TextField--success`, `TextField--warning`, `TextField--danger`, `TextField--error`), or by adding
```

Comment:
Please add the Validation states to `docs/DICTIONARIES.md` where are already others dictionaries mentioned. And here add a reference.

Reply:
done